.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+
+```yaml
+---
+group: install2
+title: Continue with your installation
+version: 2.1
+GitHub_link: install-gde/continue.md
+---
+```
+
+Refer to the following table for a description of each key-value pair.
+
+<table style="width:100%">
+   <colgroup>
+      <col width="30%">
+      <col width="70%">
+   </colgroup>
+   <thead>
+      <tr style="background-color:lightgray">
+         <th>Key-value pair</th>
+         <th>Description</th>
+      </tr>
+   </thead>
+   <tbody>
+         <td>layout: default</td>
+         <td>Selects the template Jekyll will use to render the .md file into HTML and CSS.</td>
+      <tr>
+         <td>group: install</td>
+         <td>Defines which guide the file belongs to and which left-hand menu collection the file will show up in. What you put here does not affect the top navigation menu, which is controlled by the `_/includes/navigation.html` file.</td>
+      </tr>
+      <tr>
+         <td>title: Install Magento</td>
+         <td>Sets the title of the page in the HTML meta and the main title on the page.</td>
+      </tr>
+      <tr>
+         <td>version: 2.1</td>
+         <td>Specifies which version(s) of Magento the topic affects. We also use this data to build links to the file on GitHub.</td>
+      </tr>
+      <tr>
+        <td>GitHub_link: install-gde/continue.md</td>
+        <td>Specifies the name and location of the source file in the GitHub repository.</td>
+      </tr>
+   </tbody>
+</table>
+
+### Add a Contributor's name to a topic
+When a community member contributes an entire topic—or makes substantial improvements to an existing topic—we like to thank them by adding their name (or company name) beneath the topic title and a link to their blog or website.
+
+## Report an issue {#issues}
+If you find a typo or erroneous information in Magento DevDocs, you can either fix it with a pull request (as described above) or you can report it by creating an issue in the DevDocs GitHub repository.
+GitHub issues use templates for requested information. Enter as much information as you can including content corrections, steps to reproduce, command or code updates, or questions for clarifications.
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+Check the [existing issues](https://GitHub.com/magento/devdocs/issues) on GitHub to see if someone has already reported the issue.
+</div>
+
+You have a couple of options to enter an issue:
+* Have general feedback? Create an issue on [GitHub DevDocs](https://GitHub.com/magento/devdocs/issues/new){:target="_blank"}.
+* Have feedback on a specific DevDocs page? Click the **Report an Issue** link at the top right of the page to report on the currently open topic.
+* Have a Community code contribution? Create an issue to [request DevDocs content](https://GitHub.com/magento/devdocs/issues/new?template=COMMUNITY_ISSUE_TEMPLATE.md){:target="_blank"}.
+
+
```

Comment:
Maybe add a link to our slack channel somewhere in this section:

https://magentocommeng.slack.com/messages/CAN932A3H

Reply:
Oh good point, adding some contact info